app/presenters/host_status_presenter.rb

```diff
@@ -0,0 +1,134 @@
+class HostStatusPresenter
+  class Collection < Array
+    def search_for(*search_options)
+      # TODO
```

Comment:
What is missing here?

Reply:
I forgot to delete this comment. The API does not support search and returns all statuses.